src/Microsoft.ServiceFabric.Services.Remoting/V2/FabricTransport/Runtime/FabricTransportServiceRemotingListener.cs

```diff
@@ -40,18 +40,24 @@ public class FabricTransportServiceRemotingListener : IServiceRemotingListener
         ///     for message processing.
         /// </param>
         /// <param name="serializationProvider">It is used to serialize deserialize request and response body </param>
+        /// <param name="activitySourceName">Name of the ActivitySource to be used for tracing</param>
+        /// <param name="activitySourceVersion">Version of ActivitySource to be used for tracing</param>
         /// <param name="remotingListenerSettings">The settings for the listener</param>
         public FabricTransportServiceRemotingListener(
             ServiceContext serviceContext,
             IService serviceImplementation,
             FabricTransportRemotingListenerSettings remotingListenerSettings = null,
-            IServiceRemotingMessageSerializationProvider serializationProvider = null)
+            IServiceRemotingMessageSerializationProvider serializationProvider = null,
+            string activitySourceName = "Microsoft.ServiceFabric.ActivitySource",

```

Comment:
We need not require this overload I guess. If user wants to override the activity source name, then they need to pass MessageDispatcher object with ServiceRemotingListener